src/main/java/com/nutomic/syncthingandroid/fragments/FolderPathTextView.java

```diff
@@ -0,0 +1,64 @@
+package com.nutomic.syncthingandroid.fragments;
+
+import android.app.Activity;
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.text.TextUtils;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import com.nutomic.syncthingandroid.R;
+import com.nutomic.syncthingandroid.activities.FolderPickerActivity;
+
+/**
+ * Simply displays a clickable view that allows for selecting and displaying a folder path.
+ * The selected path can be accessed by adding a textwatcher in the parent activity/fragment to fragment_directory_text_view.
+ */
+
+public class FolderPathTextView extends Fragment {
```

Comment:
Actually, I think you can just extend TextView directly for what you're doing here.